### PR TITLE
Update dotnet-trace.md

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

**Comment by Youssef1313** [comment]
I don't know if the new commands I added in this PR exist in version 3.0 or only 3.1.
If they apply only to 3.1, the article must reflect this.

File: docs/core/diagnostics/dotnet-trace.md [hunk 1]
```diff
@@ -88,18 +89,30 @@ dotnet-trace collect [-h|--help] [-p|--process-id] [--buffersize <size>] [-o|--o
...
+  List of CLR runtime events to emit.
+
+- **`--clreventlevel <clreventlevel>`**
+
+  Verbosity of CLR events to be emitted.
+
+- **`-n, --name <name>`**
+
+  The name of the process to collect the trace from.
+
```
Comment (tdykstra) [inline docs/core/diagnostics/dotnet-trace.md:107]: In the CLI ref docs we've been putting options in alphabetical order.

File: docs/core/diagnostics/dotnet-trace.md [hunk 2]
```diff
@@ -56,23 +56,24 @@ Collects a diagnostic trace from a running process.
 ### Synopsis
 
 ```console
-dotnet-trace collect [-h|--help] [-p|--process-id] [--buffersize <size>] [-o|--output]
-    [--providers] [--profile <profile-name>] [--format]
+dotnet-trace collect [-h|--help] [-p|--process-id <pid>] [--buffersize <size>] [-o|--output <trace-file-path>]
+    [--providers <list-of-comma-separated-providers>] [--profile <profile-name>] [--format <Chromium|NetTrace|Speedscope>]
+    [--clrevents <clrevents>] [--clreventlevel <clreventlevel>] [-n, --name <name>]
```
Comment (tdykstra) [inline docs/core/diagnostics/dotnet-trace.md:61]: In the CLI ref docs we've been putting options in alphabetical order.

**Review by tdykstra — COMMENTED** [review]
Looks good, just a note about the order of options.

**Comment by sdmaclea** [comment]
LGTM

@josalem Can you comment on the availability of the different options?  I expect the option availability has more to do with the version of `dotnet-trace` than of `dotnet` runtime.

**Comment by josalem** [comment]
The options for dotnet-trace are only dependent on the version of dotnet trace and _not_ the runtime.

**Review by tdykstra — APPROVED** [review]
Thanks @Youssef1313 !

**Pull request merged**